packages/cli/src/Init.ts

```diff
@@ -262,7 +262,7 @@ export class Init implements Command {
         warnings.push(
           `${chalk.yellow(
             'warn',
-          )} You already have a .gitignore. Don't forget to exclude .env to not commit any secret.`,
+          )} You already have a .gitignore. Don't forget to include .env in it to not commit any private information.`,
```

Comment:
One can argue that you have to "include" and exclusion rule, so "Don't forget to exclude .env to not commit any secret." here means add a line to your `.gitignore` file.

I'm not sure this change is better than before 🤔

Reply:
I understand the use of the word exclude, and a developer reading it would probably go through the same line of thinking as you have. I would also argue that the version with the least ambiguity is better, however, and the message's intent should be understood intuitively, which is to say with no thinking at all.